hack/e2e-apps/kubernetes.bats

```diff
@@ -60,9 +60,10 @@ spec:
       roles:
       - ingress-nginx
   storageClass: replicated
+  version: "v1.30"
 EOF
   kubectl wait namespace tenant-test --timeout=20s --for=jsonpath='{.status.phase}'=Active
-  timeout 10 sh -ec 'until kubectl get kamajicontrolplane -n tenant-test kubernetes-test; do sleep 1; done'
+  timeout 10 sh -ec 'until kubectl get kamajicontrolplane -n tenant-test kubernetes-test | grep -q "v1.30"; do sleep 1; done'
```

Comment:
This line checks that a `KamajiControlPlane` was created with a specific value in the `spec.version` field. What's needed is a check that you have a functional Kubernetes API that reports its version as the one requested.